### PR TITLE
Fixed compatibility with Symfony 3.0

### PR DESCRIPTION
**Pull request by hason**

### CODE REVIEW

**Comment by OskarStark** [comment]
can you add a test case for this @hason ?

**Comment by hason** [comment]
@OskarStark How can I add test for a template? This PR only fixed missing correction in the template for ACL (fix was introduced in https://github.com/sonata-project/SonataAdminBundle/commit/ab159e9b6f9b3dbbb51d9614da7f23f8c7c4f3b2#diff-5277d9825ebb0f8b6eff6b55c491668aL14).

**Comment by OskarStark** [comment]
sorry, no many tabs :smile: , i was in the wrong PR, my fault :+1: 

Thank you

File: Resources/views/CRUD/base_acl_macro.html.twig [hunk 1]
```diff
@@ -11,7 +11,8 @@ file that was distributed with this source code.
 
 {% macro render_form(form, permissions, td_type, admin, admin_pool, object) %}
     <form class="form-horizontal"
-          action="{{ admin.generateUrl('acl', {'id': admin.id(object), 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}" {{ form_enctype(form) }}
+          action="{{ admin.generateUrl('acl', {'id': admin.id(object), 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}"
+          {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
```
Comment (Soullivaneuh) [inline Resources/views/CRUD/base_acl_macro.html.twig]: Using `form_start` would be better.

Comment (OskarStark) [inline Resources/views/CRUD/base_acl_macro.html.twig]: but we didn't use it before...

Comment (rande) [inline Resources/views/CRUD/base_acl_macro.html.twig]: I add this local change on my computer too.

Comment (Soullivaneuh) [inline Resources/views/CRUD/base_acl_macro.html.twig]: This is the recommended way told by Symfony deprecation AFAIK.

Comment (rande) [inline Resources/views/CRUD/base_acl_macro.html.twig]: "Recommended" does not mean it match all cases ;) The sonata's form is a bit specific ...

**Comment by Soullivaneuh** [comment]
```php
$form = $this->createForm(new Type(), array(
    'action' => $admin->generateUrl([...]),
));
```

What is the specificity here? :smile:

File: Resources/views/CRUD/base_acl_macro.html.twig (hunk 1 above)
Comment (Soullivaneuh) [inline Resources/views/CRUD/base_acl_macro.html.twig]:
```php
$form = $this->createForm(new Type(), array(
    'action' => $admin->generateUrl([...]),
));
```

What is the specificity here? :smile:

**Pull request merged**